mettle/configure.ac

```diff
@@ -12,7 +12,7 @@ AM_PROG_CC_C_O
 AC_PROG_LIBTOOL
 LT_INIT
 
-CFLAGS="$CFLAGS -Wall -std=gnu99"
+CFLAGS="$CFLAGS -Wall -Werror -std=gnu99"
```

Comment:
Do we really want `-Werror` or are we sufficient with a policy to never merge mettle code with warnings?